apps/dashboard/app/models/remote_transfer.rb

```diff
@@ -0,0 +1,157 @@
+class RemoteTransfer < Transfer
+
+  validates_each :src_remote, :dest_remote do |record, _, remote|
+    remote_type = RcloneUtil.remote_type(remote)
+    if remote_type.nil? && remote != 'fs'
+      record.errors.add :base, "Remote #{remote} does not exist"
+    elsif ::Configuration.allowlist_paths.present? && (remote_type == 'local' || remote_type == 'alias')
+      record.errors.add :base, "Remotes of type #{remote_type} are not allowed due to ALLOWLIST_PATH"
+    end
```

Comment:
I'm going to defer this, but I wonder how much `allowlist_paths` make sense for remote file systems. I don't think now that these paths distinguish between local and remote so there's possibility for collision here.

Reply:
To me at least `allowlist_paths` for remote file systems does not make much sense. 
I can only think of one use case at the moment, which would be if the remote file system is only available through OOD, and one would want to limit access to it. If the user can access the remote file system in some other way, e.g. using Rclone directly, the allowlist is quite useless.

Reply:
We can keep it for now, but I've got #2233 to figure it out before release. I think I'm leaning towards your way of thinking too.

I think our use cases are more around personal buckets. I'm using my person OneDrive to test, but I suspect real life users will use _their own_ (or shared with a finite set of users) buckets/drives/etc, which doesn't make a lot of sense for an admin to set allow paths for.